<?xml version="1.0" encoding="UTF-8"?>
<!--
  ScreenLogger - A tool that regularly saves screenshots.
  Copyright (C) 2021 Lukas Boersma <mail@lukas-boersma.com>
  
  This program is free software: you can redistribute it and/or modify
  it under the terms of the GNU General Public License as published by
  the Free Software Foundation, either version 3 of the License, or
  (at your option) any later version.
  
  This program is distributed in the hope that it will be useful,
  but WITHOUT ANY WARRANTY; without even the implied warranty of
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  GNU General Public License for more details.
  
  You should have received a copy of the GNU General Public License
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
-->
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScreenLogger Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/Resources.qrc">
    <normaloff>:/ScreenLogger/icon.png</normaloff>:/ScreenLogger/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="capturingGroupBox">
     <property name="title">
      <string>Screenshot Capturing</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Image folder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="imageFolderEdit">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="imageFolderBtn">
          <property name="text">
           <string>Select folder</string>
          </property>
          <property name="icon">
           <iconset theme="BrowserReload">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Time between captures</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDoubleSpinBox" name="timeBetweenCapturesEdit">
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="timeBetweenCapturesUnitCombo">
          <property name="currentText">
           <string/>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="showImageFolderBtn">
        <property name="text">
         <string>Show Folder in Explorer</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="folderStructureFlatRadioBtn">
           <property name="text">
            <string>Everything in the same folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="folderStructureDayRadioBtn">
           <property name="text">
            <string>One subfolder for each day</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Folder structure</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="screenshotGroupBox">
     <property name="title">
      <string>Screenshot Image Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image scale</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="imageScaleEdit">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>20.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Image format</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Color mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QGroupBox" name="verticalGroupBox">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QRadioButton" name="imageFormatJpgRadioBtn">
           <property name="text">
            <string>JPEG (Compression artifacts, smaller files)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="imageFormatPngRadioBtn">
           <property name="text">
            <string>PNG (Lossless compression, larger files)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QGroupBox" name="verticalGroupBox_2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="colorModeColorRadioBtn">
           <property name="text">
            <string>Full color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="colorModeBWRadioBtn">
           <property name="text">
            <string>Black and white</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="timestampCheckBox">
        <property name="text">
         <string>Add a timestamp text with the date and time of the capture</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="exampleScreenshot">
        <property name="text">
         <string>Show me an example</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="aboutBtn">
       <property name="text">
        <string>About ScreenLogger</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dlgBtnBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>imageFolderEdit</tabstop>
  <tabstop>imageFolderBtn</tabstop>
  <tabstop>showImageFolderBtn</tabstop>
  <tabstop>timeBetweenCapturesEdit</tabstop>
  <tabstop>timeBetweenCapturesUnitCombo</tabstop>
  <tabstop>imageScaleEdit</tabstop>
  <tabstop>imageFormatJpgRadioBtn</tabstop>
  <tabstop>imageFormatPngRadioBtn</tabstop>
  <tabstop>colorModeColorRadioBtn</tabstop>
  <tabstop>colorModeBWRadioBtn</tabstop>
  <tabstop>timestampCheckBox</tabstop>
  <tabstop>aboutBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../res/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dlgBtnBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dlgBtnBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
